ampligraph/latent_features/models/ConvE.py

```diff
@@ -814,16 +809,7 @@ def _initialize_early_stopping(self):
                 logger.error(msg)
                 raise ValueError(msg)
 
-            if self.early_stopping_params['corrupt_side'] == 's+o':
```

Comment:
is it supported now? if not, we need to raise the exception.

Reply:
It's not supported, raises a ValueError if attempted. 

If it's "s,o" or "s" then it just gives a warning about performance issues as it'll be very slow.

Reply:
in that case you can merge the pull request